magefile.go

```diff
@@ -21,6 +21,8 @@
 package main
 
 import (
+	"path/filepath"
+
 	"github.com/magefile/mage/sh"
 
```

Comment:
```suggestion

func init() {
	mage.BeatLicense = "Elastic License"
	mage.BeatDescription = "Shipper processes, queues, and ships data."
}

```

You need to set the license type for the targets that automatically add the license headers to work.

Reply:
P.S. we have the Apache license headers on this code when it should be the Elastic license.